guide/tutorial-2-modification-of-the-environment-20-minutes.md

```diff
@@ -115,6 +115,9 @@ Finally add a Physics node to the `physics` field of the Solid. By modifying the
 *{0, 0.1, -0.2}* for example). Save the simulation. The result is depicted in
 [this figure](#your-first-rigid-body-in-webots).
 
+> **Note**:
```

Comment:
I would say: the simplest solution for you to have a coherent tutorial. An alternative is to change the position of reference to the image to let this match (if possible?).